eve/main.yml

```diff
@@ -2,7 +2,7 @@
 version: 0.2
 
 branches:
-  feature/*, documentation/*, improvement/*, bugfix/*, w/*, q/*, hotfix/*, dependabot/*:
+  feature/*, documentation/*, improvement/*, bugfix/*, w/*, q/*, hotfix/*, dependabot/*, user/*:
```

Comment:
should this be commited?

Reply:
yep, so everyone can create a `user/` branch to test things on the CI and not have to make `feature/` or `documentation/` branches.